openlibrary/i18n/__init__.py

```diff
@@ -13,15 +13,12 @@
 root = os.path.dirname(__file__)
 
 def _compile_translation(po, mo):
-    try:
-        catalog = read_po(open(po))
-
-        f = open(mo, 'wb')
-        write_mo(f, catalog)
-        f.close()
-        print('compiled', po, file=web.debug)
-    except:
-        print('failed to compile', po, file=web.debug)
```

Comment:
Having the failing file logged explicitly is useful information. If your main concern is that it be a hard fail, you can accomplish that by re-raising the exception.

Reply:
Good idea! Done.